python/cudf/cudf/tests/test_csv.py

```diff
@@ -979,6 +979,30 @@ def test_csv_reader_filepath_or_buffer(tmpdir, path_or_buf, src):
     assert_eq(expect, got)
 
 
+def test_small_zip(tmpdir):
+    df = pd.DataFrame(
+        {
+            "a": [1997] * 2,
+            "b": ["Ford"] * 2,
+            "c": ["Super, luxurious truck"] * 2,
+        }
+    )
+
+    fname = tmpdir.join("small_zip_file.zip")
+    df.to_csv(fname, index=False)
+
+    df = pd.DataFrame(
+        {
+            "a": [1997] * 2,
+            "b": ["Ford"] * 2,
+            "c": ["Superd, luxurious truck"] * 2,
+        }
+    )
+
```

Comment:
Not sure why df is declared here, looks like the second declaration can be removed. The test is failing because one of the field values has been modified.
```suggestion
```